src/libs/flow/flow_workspace.hpp

```diff
@@ -84,6 +84,12 @@ class FLOW_API Workspace
     /// return a string of recorded timing events
     std::string    timing_info() const;
 
+    // control over level of detail in caliper annotations
+    // detailed == true will annotate using filter names
+    // detailed == false will annotate using filter type
+    bool           detailed_annotations() const;
+    void           set_detailed_annotations(bool detailed);
```

Comment:
in the future we will wire an ascent option to this, not sure I like the current name.